### PR TITLE
Docs Update

### PR DESCRIPTION
**Pull request by sushantdhiman**
Changes

- Split documentation into `/documentation` folder. With sub docs inside this folder
- Use badge for license
- Added `.npmignore`

### CODE REVIEW

File: documentation/Readme.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Documentation
+
+## Introduction
+
+`Node-MySQL2` is aims to be a drop in replacement for [node-mysql](https://github.com/felixge/node-mysql). Please check `node-mysql` for full documentation.
```
Comment (sidorares) [inline documentation/Readme.md]: 'is aims' - is that correct grammar?

also maybe change links from node-mysql to mysqljs org?

Comment (sushantdhiman) [inline documentation/Readme.md]: 1) Its not :), removed extra is

2) done

**Comment by sidorares** [comment]
https://github.com/mysqljs/node-mysql should be https://github.com/mysqljs/mysql instead

File: documentation/Readme.md (hunk 1 above)
Comment (sidorares) [inline documentation/Readme.md]: Thanks! see my comment on PR about mysqljs link

**Comment by sushantdhiman** [comment]
Done

**Comment by sidorares** [comment]
Looks good, thanks!

**Pull request merged**